lte/gateway/c/li_agent/src/main.cpp

```diff
@@ -116,7 +116,7 @@ int main(void) {
     return -1;
   }
 
-  MLOG(MERROR) << "CleanUP " << pcap_dispatch;
+  MLOG(MERROR) << "CleanUP " << reinterpret_cast<void*>(&pcap_dispatch);
```

Comment:
Not sure why this is at error log level @koolzz should we just remove this log? Or bump it down to debug.

Reply:
Looks to me like we should, tagging @ymasmoudi  for visibility